utils/misc.py

```diff
@@ -151,3 +151,32 @@ def download_ampere_imagenet():
 
     os.environ["IMAGENET_IMG_PATH"] = str(dataset)
     os.environ["IMAGENET_LABELS_PATH"] = str(labels)
+
+
+def download_coco_dataset():
+    from utils.downloads.utils import get_downloads_path
```

Comment:
I would consider moving this function to COCO file and make it part of COCO dataset class